telematic_system/telematic_historical_data_processing/rosbag2_processing_service/rosbag2_processing_service/rosbag_processor.py

```diff
@@ -85,6 +85,14 @@ def process_rosbag(self,rosbag2_path):
         try:
             with open(rosbag2_path, "rb") as file:
                 reader = make_reader(file, decoder_factories=[DecoderFactory()])
+
+                try:
+                    # Check file validity. Below method checks for optional summary field - must be valid if not None
+                    reader.get_summary()
```

Comment:
Not sure need to check validity if we could catch the original MemoryError

Reply:
Good point